lib/page/homepage.dart

```diff
@@ -182,8 +217,15 @@ class _ShowCategoriesState extends State<ShowCategories> {
             padding: const EdgeInsets.only(bottom: CustomPadding.md),
             child: HomeContent(
                 title: 'Popular events',
-                isPopular: true,
                 titleBottomSpacing: CustomPadding.xs,
+                isPair: true,
+                secondWidget: CustomTextButton(
+                    text: 'See All >',
+                    fontSize: 14.0,
```

Comment:
fonts size should use the custom font size in theme

Reply:
ah sorry i missed that one, changed now